main/manager.c

```diff
@@ -6235,6 +6238,8 @@ static int action_originate(struct mansession *s, const struct message *m)
 		.uniqueid = astman_get_header(m, "ChannelId"),
 		.uniqueid2 = astman_get_header(m, "OtherChannelId"),
 	};
+	const char *gosub = astman_get_header(m, "PredialGoSub");
```

Comment:
`PreDialGoSub` for consistent capitalization.

Reply:
Thanks, done.